src/utils/constants/commandInfo.ts

```diff
@@ -0,0 +1,76 @@
+// Default have this for every built in command 
+export interface ICommandInformation {
+    name: string,
+    description: string,
+    contract: string
+}
+
+export function prettyPrintCommand (command: ICommandInformation) {
+    return `${command.name}\n${command.description}\ncontract: ${command.contract} \n`;
```

Comment:
How do you feel about `Usage` instead of `contract`? Contract didn't immediately convey what it was for until I looked at the examples.

Reply:
Yes I agree! Was reading some documentation earlier today and realized i made a mistake 🤦